libraries/core_libs/consensus/include/pbft/pbft_manager.hpp

```diff
@@ -465,27 +467,29 @@ class PbftManager : public std::enable_shared_from_this<PbftManager> {
   /**
    * @brief If there are enough certify votes, push the vote PBFT block in PBFT chain
    * @param cert_voted_block_hash PBFT block hash
-   * @param cert_votes_for_round certify votes
+   * @param current_round_cert_votes certify votes
    * @return true if push a new PBFT block in chain
    */
   bool pushCertVotedPbftBlockIntoChain_(blk_hash_t const &cert_voted_block_hash,
-                                        std::vector<std::shared_ptr<Vote>> &&cert_votes_for_round);
+                                        std::vector<std::shared_ptr<Vote>> &&current_round_cert_votes);
 
   /**
    * @brief Final chain executes a finalized PBFT block
    * @param sync_block PBFT block, cert votes, DAG blocks, and transactions
    * @param finalized_dag_blk_hashes DAG blocks hashes
    * @param sync it's true when it's last finalized block
    */
-  void finalize_(SyncBlock &&sync_block, std::vector<h256> &&finalized_dag_blk_hashes, bool sync = false);
+  void finalize_(PeriodData &&sync_block, std::vector<h256> &&finalized_dag_blk_hashes, bool sync = false);
 
   /**
    * @brief Push a new PBFT block into the PBFT chain
-   * @param sync_block PBFT block, cert votes, DAG blocks, and transactions
+   * @param sync_block PBFT block, cert votes for previous period, DAG blocks, and transactions
+   * @param cert_votes cert votes for pbft block period
    * @param dag_blocks_order DAG blocks hashes
    * @return true if push a new PBFT block into the PBFT chain
    */
-  bool pushPbftBlock_(SyncBlock &&sync_block, vec_blk_t &&dag_blocks_order = {});
+  bool pushPbftBlock_(PeriodData &&sync_block, std::vector<std::shared_ptr<Vote>> &&cert_votes,
```

Comment:
refactor also arg name, not just type

Reply:
not sure which thing exactly you are referring to here?